user/language_features/configuration.md

```diff
@@ -56,3 +63,38 @@ gauge_clear_state_level = spec
 ```
 This clears the objects after the execution of each specification, so that new objects are created for next execution.
 
+
+## CSharp
+
+CSharp Specific configuration changes can be made in the ***env/default/default.properties*** file.
+
+### gauge_reports_dir
+* The path to the gauge reports directory.
+* Should be either relative to the project directory or an absolute path.
+
+Example:
+```
+gauge_reports_dir = reports
+```
+
+### overwrite_reports
+* Set as false if gauge reports should not be overwritten on each execution. 
+* A new time-stamped directory will be created on each execution.
+
+Example:
+```
+overwrite_reports = true
+```
+
+### screenshot_on_failure
+Set to false to disable screenshots on failure in reports.
+
+Example:
+```
+screenshot_on_failure = true
+```
+
+
+
+## Ruby
```

Comment:
Good idea, Ill put that in.  I got the CSharp properties from the example project so I'll check the Ruby one. Otherwise ill put in the comment you suggested.